new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -398,6 +534,14 @@ public void tearDownAll() {
       logger.info("Deleted namespace: " + domainNamespace);
     }
 
+    // Delete domain namespaces
```

Comment:
we need to make sure the diagnostics are collected - i think it should be left to the extension to delete the namespaces

Reply:
yes, Sankar will make the necessary change as part of his PR for diagnstics